### ISSUE TITLE
desktop file does not pass desktop-file-validate on Ubuntu trusty

### ISSUE THREAD

**Issue by probonopd** [cgarry/ser-player#5]
The desktop file does not pass `desktop-file-validate` on Ubuntu trusty which is required for inclusion in the AppImageHub central directory of available AppImages.

```
ser-player.desktop: warning: key "Encoding" in group "Desktop Entry" is deprecated
ser-player.desktop: error: value "Video;AudioVideo" for string list key "Categories" in group "Desktop Entry" does not have a semicolon (';') as trailing character
```

I would have sent a PR but I see that the desktop file is three times in this repository:
https://github.com/cgarry/ser-player/search?utf8=%E2%9C%93&q=%22Desktop+Entry%22

Can you combine these three into one, and use `cp` commands as needed?

Anyhow, this is how it would need to be changed:

```
[Desktop Entry]
Type=Application
Name=SER Player
Name[da]=SER Afspiller
Name[fr]=Lecteur SER
Comment=A video player and processing utility for SER files
Comment[da]=En video afspiller og billedprocessing utility for SER format
Comment[de]=Video Player und Verarbeitungswerkzeug für SER-Dateien
Comment[es]=Un reproductor de Video y utilidad de procesamiento para archivos SER
Comment[fr]=Un utilitaire de lecture et de traitement de vidéos SER
Exec=ser-player %f
Icon=ser-player
MimeType=video/ser;
Terminal=false
Categories=Video;AudioVideo;
```

Thank you very much for your consideration.

Reference:
https://github.com/AppImage/appimage.github.io/pull/173

**Comment by cgarry** [comment]
This should be fixed in v1.7.2 that has just been released:
https://github.com/cgarry/ser-player/releases/download/v1.7.2/SER_Player-v1.7.2-x86_64.AppImage

**Comment by probonopd** [comment]
Thank you. Added to https://appimage.github.io/ser-player/
